apps/nerves_hub_device/lib/nerves_hub_device_web/channels/device_channel.ex

```diff
@@ -42,15 +42,18 @@ defmodule NervesHubDeviceWeb.DeviceChannel do
         Devices.resolve_update(device, deployments)
         |> build_join_reply()
 
-      if join_reply.update_available do
+      if join_reply.update_available and params[:download_uuid] == nil do
```

Comment:
```suggestion
      if join_reply.update_available and params["download_uuid"] == nil do
```

I believe `params` will be a JSON decoded map here, so it will need to check binary keys